support/mender-inventory-geo

```diff
@@ -35,27 +30,33 @@ err() {
 # This value is in seconds
 TIMEOUT=10
 
-# find the ip address
-ip=`wget --timeout=${TIMEOUT} -qO /dev/stdout --header "Content-Type: text/plain;" \
-     --header 'Host: ifconfig.io' --header 'User-Agent: curl/7.67.0' \
-     --header 'Accept: */*' ifconfig.io | tail -1`
-
-[ -n "${ip}" ] || err 2 "Unable to get the IP address from ifconfig.io"
-
-# get the information, see https://www.ipvigilante.com/api-developer-docs/
-geo=`wget --timeout=${TIMEOUT} -qO /dev/stdout https://ipvigilante.com/csv/${ip} 2>/dev/null`
-
-[ -n "${geo}" ] || err 3 "Unable to get the geolocalization data from ipvigilante.com"
+ipinfo=$(wget --timeout=${TIMEOUT} -q -O /dev/stdout \
+              --header 'Accept: application/json' \
+              https://ipinfo.io)
+if [ $? != 0 ] || [ -z "${ipinfo}" ]; then
+    err 2 "Unable to get IP info from ipinfo.io"
+fi
+# Convert JSON (Object{string -> string}) to key=value format.
+ipinfo=$(echo "${ipinfo}" | \
+             tr -d "\n" | \
```

Comment:
This one I had to stare at for a while :p 
And concluded that it is just fine, still I couldn't help myself, so I tried a little with my favourite `awk`, which gave me a PoC:
```
cat test.json | awk -F : '
length($1) > 1 {gsub("\"|,| ", "", $1); \
gsub("\"|,| ", "", $2); printf ("%s=%s\n", $1, $2)}'

Which gives me out:
```
ip=195.159.235.214
hostname=195-159-235-214.customer.powertech.no
city=Oslo
region=Oslo
country=NO
loc=59.912710.7461
org=AS2116GLOBALCONNECTAS
postal=0001
timezone=Europe/Oslo
readme=https
```
Which is also not robust, if you look at the `readme`. But still fun!

Tldr; this is just fine, I just couldn't help myself.
```